docker-client/1.0.10.wso2v1/pom.xml

```diff
@@ -0,0 +1,142 @@
+            <optional>true</optional>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <version>2.4.0</version>
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Bundle-Vendor>WSO2, Inc.</Bundle-Vendor>
+                        <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Bundle-Name>${project.artifactId}</Bundle-Name>
+                        <Export-Package>
+                            io.fabric8.docker.*;version="${project.version}";-split-package:=merge-first
+                        </Export-Package>
+                        <Private-Package>
+                        </Private-Package>
+                        <Import-Package>
+                            !io.fabric8.docker.*,
+                            com.fasterxml.jackson.annotation;version="[2.7.0,3.0.0)",
+                            com.fasterxml.jackson.databind;version="[2.7.0,3.0.0)",
+                            com.fasterxml.jackson.databind.type;version="[2.7.0,3.0.0)",
+                            com.squareup.okhttp;version="[2.7.2,3.0.0)",
+                            com.squareup.okhttp.logging;version="[2.7.2,3.0.0)",
+                            com.squareup.okhttp.ws;version="[2.7.2,3.0.0)",
+                            io.fabric8.docker.api.builder;version="[1.0.0,2.0.0)",
+                            io.fabric8.docker.api.model;version="[1.0.0,2.0.0)",
+                            io.fabric8.docker.dsl;version="[1.0.0,2.0.0)",
+                            io.fabric8.docker.dsl.container;version="[1.0.0,2.0.0)",
+                            io.fabric8.docker.dsl.image;version="[1.0.0,2.0.0)",
+                            io.fabric8.docker.dsl.misc;version="[1.0.0,2.0.0)",
+                            io.fabric8.docker.dsl.network;version="[1.0.0,2.0.0)",
+                            io.fabric8.docker.dsl.volume;version="[1.0.0,2.0.0)",
+                            javax.net,
+                            javax.net.ssl,
+                            javax.security.auth.x500,
+                            javax.validation;version="[1.1.0,2.0.0)",
+                            okio;version="[1.6.0,1.7.0)",
+                            org.apache.commons.codec.binary;version="[1.10.0,2.0.0)",
+                            org.apache.commons.compress.archivers;version="[1.9.0,2.0.0)",
+                            org.apache.commons.compress.archivers.tar;version="[1.9.0,2.0.0)",
+                            org.apache.commons.compress.compressors.bzip2;version="[1.9.0,2.0.0)",
+                            org.newsclub.net.unix;version="[2.0.0,2.1.0)",
+                            org.slf4j;version="[1.7.0,2.0.0)"
```

Comment:
Fixed in https://github.com/wso2/orbit/commit/3119064e24811ce44ddeb10f16d677a572e8aaa3